tests/tbans_tests/requests/test_webhook_request.py

```diff
@@ -60,23 +60,30 @@ def test_str(self):
         self.assertTrue('WebhookRequest(notification=' in str(message_str))
 
     def test_webhook_message(self):
-        webhook_payload = {'test': 'something'}
-        notification = MockNotification(webhook_payload=webhook_payload)
+        webhook_message_data = {'test': 'something'}
+        notification = MockNotification(webhook_message_data=webhook_message_data)
         message = WebhookRequest(notification, 'https://www.thebluealliance.com/', 'secret')
-        self.assertEqual(message.json_string(), '{"message_data": {"test": "something"}, "message_type": "verification"}')
+        self.assertEqual(message._json_string(), '{"message_data": {"test": "something"}, "message_type": "verification"}')
 
     def test_webhook_message_no_payload(self):
         notification = MockNotification()
         message = WebhookRequest(notification, 'https://www.thebluealliance.com/', 'secret')
-        self.assertEqual(message.json_string(), '{"message_type": "verification"}')
+        self.assertEqual(message._json_string(), '{"message_type": "verification"}')
 
     def test_generate_webhook_checksum(self):
-        message = WebhookRequest(MockNotification(webhook_payload={'data': 'value'}), 'https://www.thebluealliance.com/', 'secret')
-        message_json = message.json_string()
+        message = WebhookRequest(MockNotification(webhook_message_data={'data': 'value'}), 'https://www.thebluealliance.com/', 'secret')
+        message_json = message._json_string()
         self.assertEqual(message._generate_webhook_checksum(message_json), 'dbecd85ae53d221c387647085912d2107fa04cd5')
 
     def test_send(self):
-        message = WebhookRequest(MockNotification(webhook_payload={'data': 'value'}), 'https://www.thebluealliance.com/', 'secret')
+        message = WebhookRequest(MockNotification(webhook_message_data={'data': 'value'}), 'https://www.thebluealliance.com/', 'secret')
         response = message.send()
-        self.assertEqual(response.status_code, 200)
-        self.assertIsNone(response.content)
+        result = response.get_result()
+        self.assertEqual(result.status_code, 200)
+        self.assertEqual(result.content, 'Some content here')
+
+    def test_send_fail(self):
```

Comment:
It turns out we weren't hitting our validations in `_stub_webhook_request` in our `test_send` since the changes introduced in https://github.com/the-blue-alliance/the-blue-alliance/pull/2422 since we were never waiting to get the content. This adds a test that will fail if we don't hit our validations, which makes sure were hitting the validations in our other tests.

Reply:
<3